client/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import CustomerTable from "./components/CustomerTable";
+import UploadCSVForm from "./components/UploadCSVForm";
+import './css/main.css'
+
+function App() {
+  return (
+    <div>
+      <h1 style={{ textAlign: 'center' }}>Privy Table</h1>
+      <UploadCSVForm />
+      <CustomerTable />
+    </div>
```

Comment:
Once you learn react navigation, this file will be to set it up. I would recommend extracting this to it's own component, like `<HomePage />`